src/implementations/title-escrow/changeHolder.ts

```diff
@@ -0,0 +1,39 @@
+import signale from "signale";
+import { getLogger } from "../../logger";
+import { getWalletOrSigner } from "../utils/wallet";
+import { connectToTitleEscrow } from "./helpers";
+import { TitleEscrowChangeHolderCommand } from "../../commands/title-escrow/title-escrow-command.type";
+
+import { dryRunMode } from "../utils/dryRun";
+import { TransactionReceipt } from "@ethersproject/providers";
+
+const { trace } = getLogger("token-registry:changeHolder");
```

Comment:
hmm this part should we using 'title-escrow:changeHolder' instead? since the level before this is title-escrow

Reply:
👌